engine/src/test/resources/org/camunda/bpm/engine/test/api/runtime/oneTaskProcessWithStartForm.bpmn20.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
🔧 This file should probably move as well accordingly, given the test class moved. But it's a minor thing.

Reply:
I thought about it. But then, if we apply this pattern consistently we would need to also duplicate the `oneTaskProcess.bpmn20.xml`. Since none of those resources are strictly tied to the test class (i.e. they are not called `TestClass.testMethod.bpmn`), I think it's fine to have them with the rest of the more generic resources. What do you think?

Reply:
Fair enough, yeah, works for me 👍